memory_profiler.py

```diff
@@ -111,12 +143,7 @@ def ps_util_tool():
                 else 'get_memory_info'
             mem = getattr(process, meminfo_attr)()[0] / _TWO_20
             if include_children:
-                try:
-                    for p in process.children(recursive=True):
-                        mem += getattr(p, meminfo_attr)()[0] / _TWO_20
-                except psutil.NoSuchProcess:
-                    # https://github.com/fabianp/memory_profiler/issues/71
-                    pass
+                mem +=  sum(_get_child_memory(process, meminfo_attr))
```

Comment:
Unfortunately I'm not sure of the behavior of psutil with regards to linux forks; I simply used the same meminfo_attr as the summation function. If that's true then this is indeed a problem, but I'd guess that we'd have to dig into psutil to figure it out.